readme.md

```diff
@@ -105,7 +124,15 @@ Your Answer:
 Write some basic CSS that demonstrates changing a CSS property when the device width drops below 40rem.
 
 ```css
+body {
+  background-color: lemonchiffon;
+}
 
+@media (max-width: 40rem) {
+  body {
+    background-color: chucknorris;
+  }
+}
```

Comment:
:+1:

Chuck Norris has already been to Mars; that's why there are no signs of life.